src/components/SatelliteManifestPanel/SatelliteManifestPanel.tsx

```diff
@@ -304,43 +341,85 @@ const SatelliteManifestPanel: FunctionComponent<SatelliteManifestPanelProps> = (
                   </FlexItem>
                   <FlexItem align={{ default: 'alignRight' }}>{pagination()}</FlexItem>
                 </Flex>
-                <Table
-                  aria-label="Satellite Manifest Table"
-                  cells={getTableHeaders(user)}
-                  rows={
-                    isFetching
-                      ? []
-                      : getRowsWithAllocationDetails(
-                          data,
-                          user,
-                          searchValue,
-                          page,
-                          perPage,
-                          rowExpandedStatus,
-                          handleRowManifestClick,
-                          entitlementsRowRefs,
-                          sortBy
-                        )
-                  }
-                  onCollapse={toggleAllocationDetails}
-                  sortBy={sortBy}
-                  onSort={handleSort}
-                  actions={actions()}
-                  areActionsDisabled={() => isLoadingManifestExport}
-                >
-                  <TableHeader />
-                  <TableBody />
-                </Table>
+              </PageSection>
+              <TableComposable
+                aria-label="Satellite Manifest Table"
+                variant="compact"
+                ouiaId="manifestTable"
+                ouiaSafe={true}
+              >
+                <Thead>
+                  <Tr ouiaId="manifestTable/head" ouiaSafe={true}>
+                    <Th />
+                    {getTableHeaders(user).map((header, index) => (
+                      <Th key={index} sort={getSortParams(index)}>
+                        {header}
+                      </Th>
+                    ))}
+                    <Td />
+                  </Tr>
+                </Thead>
+                {getRows().map((row, index) => {
+                  const name = row.cells[0];
```

Comment:
Same thing here - if these were keyed instead of indexed you may not have to assign them to variables even

Reply:
That one is tougher because the column sorting of the rows is based on the index of those values in the array.  I'll look into options because again I agree it would be better.

Reply:
Okay, I think I have it working.  Had to refactor lots of stuff.  Figured out I had forgotten to hook up the correct onSort function anyway while doing this, so our sort wasn't quite working right before this refactor.

Reply:
Sweet - I'll give it a look